.github/workflows/build-linux-clang-x64-dynamic.yml

```
@@ -33,10 +33,33 @@ jobs:
               -D CMAKE_CXX_COMPILER=clang++ \
               -D BUILD_USE_PCH=ON \
               -D BUILD_INCLUDE_SYMLINK=ON \
+              -D BUILD_OPT_PROFILE=Production \
+              -D USE_TK=OFF \
               -D USE_FREETYPE=OFF \
               -D CMAKE_BUILD_TYPE=Release ..
 
     - name: Build OCCT
       run: |
```

actionlint

shellcheck reported issue in this script: SC2046:warning:2:8: Quote this to prevent word splitting

(shellcheck)